graphql-proxy/src/utils/objectToCamelCase.ts

```diff
@@ -0,0 +1,27 @@
+import memoize from "lodash/memoize";
+
+import toCamelCase from "./toCamelCase";
+
+const memoizedCamelCase = memoize(toCamelCase);
+
+const objectToCamelCase = value => {
```

Comment:
An explanation of what and why could be helpful for a future developer joining the project